src/components/ChooseCard.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+import './ChooseCard.css';
+
+class ChooseCard extends Component {
+    constructor(props) {
+        super(props)
+        this.state={
+            image : props.image,
+            name: props.name,
+            type: props.type,
+            description: props.desc,
+            atks:props.atks,
+            pv:props.pv
+        }
+    }
```

Comment:
You don't need a state nor putting props in the state, you can use props directly with `this.props.myProp` in class components.